<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cg3::viewer::MainWindow</class>
 <widget class="QMainWindow" name="cg3::viewer::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="cg3::viewer::GLCanvas" name="glCanvas" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="console"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Rendering</string>
    </property>
    <addaction name="actionChange_Background_Color"/>
    <addaction name="actionFit_Scene"/>
    <addaction name="actionReset_Point_of_View"/>
    <addaction name="actionPerspective_Orthographic_Camera_Mode"/>
    <addaction name="actionShow_Axis"/>
    <addaction name="actionShow_Unit_Box"/>
    <addaction name="actionUpdate_Canvas"/>
    <addaction name="separator"/>
    <addaction name="action2D_Mode"/>
    <addaction name="action3D_Mode"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionFull_Screen"/>
    <addaction name="actionShow_Hide_Dock_Widget"/>
    <addaction name="actionShow_Hide_Console"/>
    <addaction name="actionShow_Hide_DrawList"/>
   </widget>
   <widget class="QMenu" name="menuFile_2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLoad_Mesh"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Point_of_View"/>
    <addaction name="actionLoad_Point_Of_View_from"/>
    <addaction name="actionSave_Point_Of_View"/>
    <addaction name="actionSave_Point_Of_View_as"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Snapshot"/>
   </widget>
   <widget class="QMenu" name="menuDebug_Objects">
    <property name="title">
     <string>Debug Objects</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionToggle_Debug_Objects"/>
   </widget>
   <addaction name="menuFile_2"/>
   <addaction name="menuFile"/>
   <addaction name="menuDebug_Objects"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QDockWidget" name="dockToolBox">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>138</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>280</width>
          <height>508</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Page 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockDrawList">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>250</width>
          <height>541</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSave_Snapshot">
   <property name="text">
    <string>Save Snapshot</string>
   </property>
  </action>
  <action name="actionShow_Axis">
   <property name="text">
    <string>Show Axis</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionUpdate_Canvas">
   <property name="text">
    <string>Update Canvas</string>
   </property>
   <property name="shortcut">
    <string>U</string>
   </property>
  </action>
  <action name="actionFit_Scene">
   <property name="text">
    <string>Fit Scene</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionChange_Background_Color">
   <property name="text">
    <string>Change Background Color</string>
   </property>
  </action>
  <action name="actionSave_Point_Of_View">
   <property name="text">
    <string>Save Point Of View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionLoad_Point_of_View">
   <property name="text">
    <string>Load Point Of View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionShow_Hide_Dock_Widget">
   <property name="text">
    <string>Show/Hide Dock Widget</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionLoad_Point_Of_View_from">
   <property name="text">
    <string>Load Point Of View from...</string>
   </property>
  </action>
  <action name="actionSave_Point_Of_View_as">
   <property name="text">
    <string>Save Point Of View as...</string>
   </property>
  </action>
  <action name="actionShow_Hide_Console">
   <property name="text">
    <string>Show/Hide Console</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="action2D_Mode">
   <property name="text">
    <string>2D Mode</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="action3D_Mode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>3D Mode</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionReset_Point_of_View">
   <property name="text">
    <string>Reset Point of View</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionPerspective_Orthographic_Camera_Mode">
   <property name="text">
    <string>Perspective/Orthographic Camera Mode</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionLoad_Mesh">
   <property name="text">
    <string>Load Mesh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionToggle_Debug_Objects">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle Debug Objects</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionShow_Unit_Box">
   <property name="text">
    <string>Show Unit Box</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionShow_Hide_DrawList">
   <property name="text">
    <string>Show/Hide DrawList</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>cg3::viewer::GLCanvas</class>
   <extends>QWidget</extends>
   <header>cg3/viewer/glcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
